src/main/java/com/google/sps/servlets/CreateEventServlet.java

```diff
@@ -23,12 +23,13 @@
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
         KeyFactory keyFactory = datastore.newKeyFactory().setKind("event");
+        float price = request.getParameter("price");
+        long date = request.getParameter("date");
```

Comment:
I assume these values are sent as float and long and no further processing is needed. If not this will create problems.